.travis.yml

```diff
@@ -29,9 +26,6 @@ matrix:
     -
       env: PUPPET_GEM_VERSION="~> 5.0" CHECK=parallel_spec
       rvm: 2.4.4
-    -
-      env: PUPPET_GEM_VERSION="~> 4.0" CHECK=parallel_spec RUBYGEMS_VERSION=2.7.8 BUNDLER_VERSION=1.17.3
```

Comment:
Same question as with cron, do we want to drop testing against puppet4?

Reply:
I think it's fine (duplicating here for completeness)